src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1409,10 +1409,18 @@ public void TestRound()
         {
             using (CultureHelper.SetCulture("en-GB"))
             {
-                Helper.AssertTemplateResult("1.235", "{{ 1.234678 | round:3 }}");
+                Helper.AssertTemplateResult("1.235", "{{ 1.234678 | round: 3 }}");
+                Helper.AssertTemplateResult("1.23", "{{ 1.234678 | round: 2.7 }}");
+                Helper.AssertTemplateResult("1.235", "{{ 1.234678 | round: 3.1 }}");
                 Helper.AssertTemplateResult("1", "{{ 1 | round }}");
-
-                Assert.That(StandardFilters.Round("1.2345678", "two"), Is.Null);
+                Helper.AssertTemplateResult("1", "{{ 1.234678 | round }}");
+                Helper.AssertTemplateResult("1", "{{ 1.234678 | round: -3 }}");
+                Helper.AssertTemplateResult("1", "{{ 1.234678 | round: nonesuch }}");
+                Helper.AssertTemplateResult("0", "{{ nonesuch | round }}");
+
+                Assert.That(StandardFilters.Round(_contextV20, "1.2345678", "two"), Is.EqualTo(1m));

```

Comment:
This kind of change may have unintended consequences, e.g. breaking DotLiquid consumers who have built a dependency on the output.

Reply:
Right, I picked this example too fast - string for number input was never valid.